src/main/java/org/elasticsearch/client/IndicesAdminClient.java

```diff
@@ -588,6 +588,21 @@
     AnalyzeRequestBuilder prepareAnalyze(String text);
 
     /**
+     * Analyze texts under the provided index.
+     *
+     * @param index The index name
+     * @param text  The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyzeWithIndexAndMultiValued(@Nullable String index, String... text);
+
+    /**
+     * Analyze texts.
+     *
+     * @param text The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyzeWithMultiValued(String... text);
```

Comment:
I think these two methods go a bit against the naming convention that we have here for prepare* methods. I really think we should go for one of the options I mentioned [here](https://github.com/elastic/elasticsearch/pull/10847#discussion_r29516497). What do you think @dakrone ?